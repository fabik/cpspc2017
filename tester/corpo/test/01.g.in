100
55 41 49 8 71 81 87 80 79 40 59 86 4 53 67 6 65 72 2 91 28 78 95 88 94 93 16 23 100 66 74 54 39 12 20 90 17 32 68 60 99 46 42 34 7 11 21 84 92 1 14 9 38 75 37 58 50 57 73 56 43 3 62 64 36 25 24 52 13 22 15 69 27 26 45 98 30 19 35 48 5 47 82 89 76 33 51 63 83 61 29 96 77 10 85 18 44 70 31
75 60 84 87 22 48 33 35 19 68 57 76 49 9 25 71 64 15 55 61 16 30 10 51 77 96 67 78 31 36 27 38 32 59 2 83 6 74 24 73 70 7 99 39 81 40 91 82 29 18 4 80 17 26 90 50 92 58 69 37 46 41 43 62 20 88 95 14 86 28 53 52 93 94 34 44 100 72 12 21 8 11 5 63 45 54 1 3 47 56 23 85 98 79 89 65 42 13 66
