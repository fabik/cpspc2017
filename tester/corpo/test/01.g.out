99 47 36 79 83 45 7 42 29 23 9 16 29 39 27 47 89 1 25 23 5 46 39 21 48 25 33 93 19 41 43 32 43 31 31 45 7 44 63 19 73 55 5 27 61 3 17 9 40 18 49 77 14 87 24 13 17 35 33 37 6 2 71 3 1 22 95 10 38 57 91 13 75 65 49 30 26 20 12 15 4 41 8 81 69 59 85 97 35 15 11 37 67 51 11 34 0 53 28 21 
